team-members/iryna-yehorova.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>About me</title>
+  <meta charset="utf-8">
+</head>
+
+<body>
+  <header>
+    <nav>
+      <h2>Welcome to my site</h2>
+      <ul>
+        <li> <a href="#">Home</a></li>
+        <li><a href="#about_me">About me</a></li>
+        <li><a href="#abilities">My abilities</a></li>
+        <li><a href="#town">My town</a></li>
+        <li><a href="#contact">Contact me</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <h1 id="about_me">About me</h1>
+    <section>
+      <p>My name is Iryna. Here is my photo:</p>
+      <img src="../images/team-members/iryna-yehorova.jpg" alt="Iryna's photo" width="15%">
+      <br>
+    </section>
+    <section>
+      <h3>Top 5 films I like:</h3>
+      <ol>
+        <li>Harry Potter</li>
+        <li>Lord of the Rings</li>
+        <li>Tomb Raider</li>
+        <li>Avengers</li>
+        <li>Iron Man I</li>
+      </ol>
+      <p>I like many films, but I cannot remember all the names at once. </p>
+      <h3>Top 5 places I want to visit:</h3>
+      <ul>
+        <li>England</li>
+        <li>Germany</li>
+        <li>Italy</li>
+        <li>India</li>
+        <li>Australia</li>
+      </ul>
+      <p>I like my home, and I do not want to travel all over the world. I hope
+        I can visit a lot of places across whole my life.</p>
+      <br>
+    </section>
+    <section>
+      <h3 id="abilities">My abilities</h3>
+      <table>
+        <thead>
+          <tr>
+            <th>Ability</th>
+            <th>Level</th>
+            <th>Do I like it?</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>Sleeping 20 hours a day</td>
+            <td>10</td>
+            <td>No</td>
+          </tr>
+          <tr>
+            <td>Drinking tea</td>
+            <td>14</td>
+            <td>Yes</td>
+          </tr>
+          <tr>
+            <td>Working all night long</td>
+            <td>7</td>
+            <td>No</td>
+          </tr>
+          <tr>
+            <td>Reading horoscopes</td>
+            <td>52</td>
+            <td>No</td>
+          </tr>
+          <tr>
+            <td>Being kind and cooperative</td>
+            <td>24</td>
+            <td>Yes</td>
+          </tr>
+          <tr>
+            <td>Listening to music while walking</td>
+            <td>52</td>
+            <td>No</td>
+          </tr>
+        </tbody>
+      </table>
+      <p>This is not the whole list of my hidden talents. I like shopping,
```

Comment:
it is better from content to start from the new line and with the appropriate indent
review your file for the same issue and correct, pls)